xclim/indices.py

```diff
@@ -83,23 +93,27 @@ def base_flow_index(q, freq='YS'):
 @with_attrs(standard_name='cold_spell_duration_index', long_name='', units='days')
 @valid_daily_min_temperature
 def cold_spell_duration_index(tasmin, tn10, freq='YS'):
-    """Cold spell duration index.
+    r"""Cold spell duration index
 
-    Resample the daily minimum temperature series by returning the number of days per
+    Re-samples the daily minimum temperature series by returning the number of days per
     period where the temperature is below the calendar day 10th percentile (calculated
     over a centered 5-day window for values during the 1961–1990 period) for a minimum of
     at least six consecutive days.
 
     Parameters
     ----------
     tasmin : xarray.DataArray
-      Minimum daily temperature values [K].
+      Minimum daily temperature values [C] or [K]
```

Comment:
We have still no explicit unit conversion routine. That might be covered by ocgis, but I think it can get confusing if the function's doc assumes that it's being wrapped by an external library. I suggest we stick with one unit and provide complimentary unit conversion through ocgis.

Reply:
I see you assumed the unit choice everywhere. Let's keep it that way then and get unit conversion to work with or without ocgis.

Reply:
Sounds good. I'll add a TODO at the top concerning that problem.